static/js/components/Source.jsx

```diff
@@ -537,6 +538,31 @@ const SourceContent = ({ source }) => {
           </AccordionDetails>
         </Accordion>
       </Grid>
+      <Grid item xs={12} lg={6} order={{ md: 4, lg: 3 }}>
```

Comment:
4 and 3 seem pretty high up on the source page for this feature, is that intended?

Reply:
@Theodlz Not necessarily. I think it could be moved elsewhere. It was just flagged as an issue at the MMA call that the notes were not made (highly) visible on the source page.